PointDrivePlannerROS:

  drive_mode_theta_z_threshold: 0.044
  theta_z_tolerance: 0.005
  pos_tolerance: 0.01
  plan_window_dist_threshold: 0.3
